src/05-objects.js

```diff
@@ -35,7 +44,12 @@ function createObjectWithValue(val) {}
  *  createObjectWithKey("left");
  *  //> { left: false }
  */
-function createObjectWithKey(key) {}
+function createObjectWithKey(key) {
+  let obj = {};
+  obj[key] = {};
```

Comment:
i know we could set this equal to anything, but probably more appropriate in this case to set the value as null.